### PR TITLE
feat: improve on admin ui funding tab

### PR DESCRIPTION
**Pull request by dni**
now only shows settings of selected fundingsource.

also refactors fundingsources into vue component for reuse in install wizard https://github.com/lnbits/lnbits/pull/1977

screenshot-1696454913

### CODE REVIEW

File: lnbits/static/js/components.js [hunk 1]
```diff
@@ -568,3 +568,169 @@ Vue.component('lnbits-notifications-btn', {
     }
   }
 })
+
+Vue.component('lnbits-funding-sources', {
+  mixins: [windowMixin],
+  props: ['form-data', 'allowed-funding-sources'],
+  computed: {
+    funding_sources() {
```
Comment (motorina0) [inline lnbits/static/js/components.js:576]: use camel case in JS

File: lnbits/static/js/components.js [hunk 2]
```diff
@@ -568,3 +568,169 @@ Vue.component('lnbits-notifications-btn', {
...
+      let tmp = []
+      for (let [key, _, obj] of this.raw_funding_sources) {
```
Comment (motorina0) [inline lnbits/static/js/components.js:578]:
```suggestion
      for (const [key, _, obj] of this.raw_funding_sources) {
```

File: lnbits/static/js/components.js [hunk 3]
```diff
@@ -568,3 +568,169 @@ Vue.component('lnbits-notifications-btn', {
...
+        let tmp_obj = {}
```
Comment (motorina0) [inline lnbits/static/js/components.js:579]:
```suggestion
        const tmpObj = {}
```

**Comment by dni** [comment]
merge this first https://github.com/lnbits/lnbits/pull/1990

**Comment by prusnak** [comment]
> merge this first #1990

merged

**Review by talvasconcelos — APPROVED** [review]

File: lnbits/core/templates/admin/_tab_funding.html [hunk 4]
```diff
@@ -25,117 +25,70 @@ <h6 class="q-my-none">Wallets Management</h6>
...
+          ></q-toggle>
+          <br />
+          <q-toggle
+            v-if="formData.lnbits_node_ui"
+            label="Transactions Tab (Disable on large CLN nodes)"
+            v-model="formData.lnbits_node_ui_transactions"
+          ></q-toggle>
           <p v-else>Node Management not supported by active funding source</p>
         </div>
+        {% endif %}
```
Comment (jackstar12) [inline lnbits/core/templates/admin/_tab_funding.html:48]: this doesnt behave the same as before. the the `p` should show up when theres no node management available, so you'd need `{% else %}

**Review by jackstar12 — CHANGES_REQUESTED** [review]

File: lnbits/core/templates/admin/_tab_funding.html (hunk 4 above)
Comment (dni) [reply lnbits/core/templates/admin/_tab_funding.html]: good catch! missed that, thanks!

**Review by jackstar12 — APPROVED** [review]

**Review by motorina0 — APPROVED** [review]

**Pull request merged**